### ISSUE TITLE
Get test folder setup on catscan

### ISSUE THREAD

**Issue by JesseLivezey** [BouchardLab/nsds_lab_to_nwb#36]

**Comment by jihyunbak** [comment]
@JesseLivezey @jthermiz Which folder(s) on catscan are you using to pull the inputs from?

I am currently using
```bash
NSDS_DATA_PATH='/clusterfs/NSDS_data/hackathon20201201/TTankBackup/'
NSDS_STIMULI_PATH='/clusterfs/NSDS_data/hackathon20201201/Stimulus/'
```
as my test data and stimuli paths, but I don't see John's .mat file renaming (as mentioned in [this PR discussion](https://github.com/BouchardLab/nsds_lab_to_nwb/pull/45#discussion_r650258629_)) in this stimulus folder.

**Comment by JesseLivezey** [comment]
@jihyunbak I've been testing things locally on 2 blocks I copied off catscan.

It looks like 
```bash
export NSDS_STIMULI_PATH="/clusterfs/NSDS_data/stimuli"
```
should work on catscan now. For the datapath, sometime soon, we should select a (mostly) fixed set of blocks to test against and move them into a specific read-only folder on catscan. That could be `/clusterfs/NSDS_data/raw` or maybe we should have a specific testing folder and `raw` is more of a workspace.

**Comment by jihyunbak** [comment]
Addressed by #59
